python/packages/ai/teams/ai/clients/llm_client.py

```diff
@@ -284,7 +291,20 @@ def chunk_received(
 
             if streamer is not None:
                 await streamer.end_stream()
+                
```

Comment:
need to also remove L281-283 (check for `is_streaming and res.status == "success"`)